### DISCUSSION TITLE
Use docker to run and raise exception : can't evaluate field Lastmod in type page.Site

### DISCUSSION THREAD

**Discussion by MrMoreOrLess** [CaiJimmy/hugo-theme-stack#995] [Q&A]
**docker run --rm -it -p 1313:1313 -v $(pwd)/src:/src klakegg/hugo:ext-alpine server**

Start building sites … 
hugo v0.111.3-5d4eb5154e1fed125ca8e9b5a0315c4180dab192+extended linux/amd64 BuildDate=2023-03-12T11:40:50Z VendorInfo=hugoguru

ERROR 2024/03/27 15:44:27 render of "home" failed: "/src/themes/hugo-theme-stack/layouts/_default/baseof.html:4:12": execute of template failed: template: index.html:4:12: executing "index.html" at <partial "head/head.html" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/head.html:15:4": execute of template failed: template: partials/head/head.html:15:4: executing "partials/head/head.html" at <partial "head/opengraph/include.html" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/opengraph/include.html:1:3": execute of template failed: template: partials/head/opengraph/include.html:1:3: executing "partials/head/opengraph/include.html" at <partial "head/opengraph/provider/base" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/opengraph/provider/base.html:35:20": execute of template failed: template: partials/head/opengraph/provider/base.html:35:20: executing "partials/head/opengraph/provider/base.html" at <.Site.Lastmod.IsZero>: can't evaluate field Lastmod in type page.Site

ERROR 2024/03/27 15:44:27 render of "taxonomy" failed: "/src/themes/hugo-theme-stack/layouts/_default/baseof.html:4:12": execute of template failed: template: _default/list.html:4:12: executing "_default/list.html" at <partial "head/head.html" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/head.html:15:4": execute of template failed: template: partials/head/head.html:15:4: executing "partials/head/head.html" at <partial "head/opengraph/include.html" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/opengraph/include.html:1:3": execute of template failed: template: partials/head/opengraph/include.html:1:3: executing "partials/head/opengraph/include.html" at <partial "head/opengraph/provider/base" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/opengraph/provider/base.html:35:20": execute of template failed: template: partials/head/opengraph/provider/base.html:35:20: executing "partials/head/opengraph/provider/base.html" at <.Site.Lastmod.IsZero>: can't evaluate field Lastmod in type page.Site
Error: Error building site: failed to render pages: render of "taxonomy" failed: "/src/themes/hugo-theme-stack/layouts/_default/baseof.html:4:12": execute of template failed: template: _default/list.html:4:12: executing "_default/list.html" at <partial "head/head.html" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/head.html:15:4": execute of template failed: template: partials/head/head.html:15:4: executing "partials/head/head.html" at <partial "head/opengraph/include.html" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/opengraph/include.html:1:3": execute of template failed: template: partials/head/opengraph/include.html:1:3: executing "partials/head/opengraph/include.html" at <partial "head/opengraph/provider/base" .>: error calling partial: "/src/themes/hugo-theme-stack/layouts/partials/head/opengraph/provider/base.html:35:20": execute of template failed: template: partials/head/opengraph/provider/base.html:35:20: executing "partials/head/opengraph/provider/base.html" at <.Site.Lastmod.IsZero>: can't evaluate field Lastmod in type page.Site
Built in 73 ms

Site object should has Lastmod （[https://gohugo.io/methods/site/lastmod/](https://gohugo.io/methods/site/lastmod/)）but why raise these exception
Anyone know what's wrong with than? Thanks.

**Comment by CaiJimmy** [comment]
Use the latest version of Hugo, you are using v0.111.3

Duplicate https://github.com/CaiJimmy/hugo-theme-stack/issues/974

**Comment by MrMoreOrLess** [comment]
I'm sorry for wasting your time to answer this question. I should be able to find it in the issues. I just comment on this code later and
 It work, but I don't know what this code does.

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Jimmy Cai ***@***.***>
发送时间: Thursday, March 28, 2024 12:00:35 AM
收件人: CaiJimmy/hugo-theme-stack ***@***.***>
抄送: MrMoreOrLess ***@***.***>; Author ***@***.***>
主题: Re: [CaiJimmy/hugo-theme-stack] Use docker to run and raise exception : can't evaluate field Lastmod in type page.Site (Discussion #995)


Use the latest version of Hugo, you are using v0.111.3

Duplicate #974<https://github.com/CaiJimmy/hugo-theme-stack/issues/974>

―
